python/mxnet/gluon/block.py

```diff
@@ -1243,6 +1308,7 @@ def hybridize(self, active=True, backend=None, backend_opts=None, clear=True, **
             self._backend_opts = backend_opts
 
         self._active = active
+        self._partition_if_dynamic = partition_if_dynamic
         self._flags = list(kwargs.items())
         if clear:
             self._clear_cached_op()
```

Comment:
dont we need to clear `_cached_op_args` here too?

Reply:
clear `_cached_op_args` is now included in `_clear_cached_op()`